src/settings/storeconf.h

```diff
@@ -22,8 +22,9 @@ class StoreConf : public QWidget
     QString applicationName         = "Pair Storm";
     QString analizerStyle           = "white";
     QString analizerFontSize        = "12";
-    QString analizerFontName        = "Times New Roman";
-    QString cppExtentions           = "c;cpp;h;hpp";
+    QString analizerFontName        = "Courier";
+    QString cppExtentions           = "c;cpp;h;hpp;json;txt";
+    QStringList cppExtentionsList {"c", "cpp", "h", "hpp", "json", "txt"};
 
```

Comment:
I would suggest to add method getExtensions to StoreConf class in order to simplify interface

Reply:
The purpose of this class is to read data from configuration file
 and save it in persistent platform-independent application settings.
 Reach out these data are possible from any point in application using statements:
QSettings settings;
int i = settings.value("fieldName").toString();
where <fieldName> same as in configuration file.